django/db/models/sql/compiler.py

```diff
@@ -355,19 +356,18 @@ def compile(self, node, select_format=False):
             sql, params = vendor_impl(self, self.connection)
         else:
             sql, params = node.as_sql(self, self.connection)
-        if select_format and not self.subquery:
+        if select_format is FORCE or (select_format and not self.query.subquery):
```

Comment:
Where do we need to distinguish between select_format is FORCE and select_format is True?